novawallet/Modules/AssetsSearch/AssetOperation/AssetOperationNetworkList/Model/AssetOperationNetworkBuilder.swift

```diff
@@ -0,0 +1,137 @@
+import Foundation
+import Operation_iOS
+import BigInt
+
+struct AssetOperationNetworkBuilderResult {
+    let assets: [AssetListAssetModel]
+    let prices: [ChainAssetId: PriceData]
+    let state: AssetListState
+}
+
+class AssetOperationNetworkBuilder: AnyCancellableCleaning {
+    let workingQueue: DispatchQueue
+    let callbackQueue: DispatchQueue
+    let operationQueue: OperationQueue
```

Comment:
I think having both sync `operation queue` and `working queue`. Would just go with the approach we have in AssetListBuilder that uses sync `working queue`.